cancer/rank_model/cancer_rank_model_-v1.2-.ini

```diff
@@ -0,0 +1,454 @@
+    score = 5
+    priority = 4
+    string = 'incomplete_terminal_codon_variant'
+
+  [[non_coding_transcript_exon_variant]]
+    score = 3
+    priority = 2
+    string = 'non_coding_transcript_exon_variant'
+
+  [[synonymous_variant]]
+    score = 2
+    priority = 2
+    string = 'synonymous_variant'
+
+  [[mature_mirna_variant]]
+    score = 1
+    priority = 2
+    string = 'mature_mirna_variant'
+
+  [[non_coding_transcript_variant]]
+    score = 1
+    priority = 2
+    string = 'non_coding_transcript_variant'
+
+  [[regulatory_region_variant]]
+    score = 1
+    priority = 2
+    string = 'regulatory_region_variant'
+
+  [[upstream_gene_variant]]
+    score = 1
+    priority = 2
+    string = 'upstream_gene_variant'
+
+  [[regulatory_region_amplification]]
+    score = 1
+    priority = 2
+    string = 'regulatory_region_amplification'
+
+  [[tfbs_amplification]]
+    score = 1
+    priority = 2
+    string = 'tfbs_amplification'
+
+  [[5_prime_utr_variant]]
+    score = 1
+    priority = 2
+    string = '5_prime_utr_variant'
+
+  [[intron_variant]]
+    score = 1
+    priority = 2
+    string = 'intron_variant'
+
+  [[3_prime_utr_variant]]
+    score = 1
+    priority = 2
+    string = '3_prime_utr_variant'
+
+  [[feature_truncation]]
+    score = 1
+    priority = 2
+    string = 'feature_truncation'
+
+  [[TF_binding_site_variant]]
+    score = 1
+    priority = 2
+    string = 'TF_binding_site_variant'
+
+  [[stop_retained_variant]]
+    score = 1
+    priority = 2
+    string = 'stop_retained_variant'
+
+  [[feature_elongation]]
+    score = 1
+    priority = 2
+    string = 'feature_elongation'
+
+  [[regulatory_region_ablation]]
+    score = 1
+    priority = 2
+    string = 'regulatory_region_ablation'
+
+  [[tfbs_ablation]]
+    score = 1
+    priority = 2
+    string = 'tfbs_ablation'
+
+  [[coding_sequence_variant]]
+    score = 1
+    priority = 2
+    string = 'coding_sequence_variant'
+
+  [[downstream_gene_variant]]
+    score = 1
+    priority = 2
+    string = 'downstream_gene_variant'
+
+  [[NMD_transcript_variant]]
+    score = 1
+    priority = 2
+    string = 'NMD_transcript_variant'
+
+  [[intergenic_variant]]
+    score = 0
+    priority = 0
+    string = 'intergenic_variant'
+
+  [[not_reported]]
+    score = 0
+
+[filter]
+  category = variant_call_quality_filter
+  data_type = string
+  description = The filters for the variant
+  field = FILTER
+  record_rule = min
+  separators = ';',
+
+  [[not_reported]]
+    score = 0
+
+  [[pass]]
+    score = 3
+    priority = 1
+    string = 'PASS'
+
+[coverage]
+  category = variant_call_quality_filter
+  data_type = integer
+  description = Coverage on position
+  field = INFO
+  info_key = DP
+  record_rule = min
+  separators = ',',
+
+  [[not_reported]]
+    score = 0
+
+  [[low]]
+    score = -6
+    priority = 1
+    upper = 100
+    lower = -1
+
+  [[high]]
+    score = 0
+    priority = 1
+    upper = 10000000
+    lower = 100
+
+[mapping_quality]
```

Comment:
BALSAMIC already filters low MQ variants and removes their PASS filter. Can we remove this please?

Reply:
Sure, in particular if it anyway differs for different callers. Just remember that keeping fuzzy edges when there is reason for doubt instead of hard cutoffs can be beneficial!

Reply:
I totally see your point. I'm worried that it might be redundant. Now that we have a rank model, whenever we see that variants are not popping up higher because of MQ/DP/AF, we can always add them.